CONTRIBUTING.md

```diff
@@ -20,3 +20,27 @@ Git branches and their purpose and status at the time of this writing are listed
 ## Prerequisites
 
 Follow the instructions in [README.rst](https://github.com/Pylons/pyramid_openapi3/) to install the tools needed to run the project.
+
+## Testing oldest supported versions
+
+In CI, we want to test the oldest supported versions of `openapi-core` and `pyramid` on the oldest supported Python version. We do it like so:
+
+* Have the `pyproject_py37.toml` and `poetry_py37.lock` files in this repo.
+* In the Python 3.7 CI run, `mv` them to `pyproject.toml` and `poetry.lock`.
+
+### How to prepare these two files
+
+1. In `pyproject.toml` modify the `openapi-core` and `pyramid` dependencies to be pinned to the
+    oldest supported version.
+
+    ```diff
+    - openapi-core = ">=0.13.4,<0.14"
+    - pyramid = ">=1.10.7"
+    + openapi-core = "==0.13.4"
+    + pyramid = "==1.10.7"
+
+1. Re-lock `poetry.lock` using `nix-shell shell_py37.nix --run "poetry lock --no-update"`.
+1. `mv pyproject.toml pyproject_py37.toml`.
+1. `mv poetry.lock poetry_py37.lock`.
+1. `git add pyproject_py37.toml poetry_py37.lock && git commit`.
+1. `git checkout pyproject.toml poetry.lock.
```

Comment:
```suggestion
1. `git checkout pyproject.toml poetry.lock`.
```

Reply:
Up to you, but I would have put all the above into a single code block with comments to explain as needed, instead of an enumerated list. Then you don't need all the extra markup, remove the `.` at the end of each list item, and it is easier to copy-pasta.

Reply:
Thanks!